sql/derived_tables/loans_items.sql

```diff
@@ -73,6 +81,7 @@ FROM
     LEFT JOIN public.user_groups AS ug ON cl.patron_group_id_at_checkout = ug.id
     LEFT JOIN public.inventory_locations AS iel ON ii.effective_location_id = iel.id
     LEFT JOIN public.inventory_locations AS ipl ON json_extract_path_text(ii.data, 'permanentLocationId') = ipl.id
+    LEFT JOIN folio_reporting.locations_libraries AS ll ON ipl.id = ll.location_id
```

Comment:
Should this dependency be added to `deps.txt`?

Reply:
Yes, thanks! New commit updates deps.txt and runlist.txt.